ArduCopter/system.cpp

```diff
@@ -252,6 +251,14 @@ void Copter::init_ardupilot()
         enable_motor_output();
     }
 
+    if(g.planck_nav_lights_on)
+    {
+      notify.set_nav_lights();
```

Comment:
Do you have to call all 4 individually 4 times, or sequentially call each one 4 times (as written here)?

Regardless, please make a comment detailing that this appears to be necessary

Reply:
I don't know if its necessary to call each sequentially, or if writing to each LED 4 times before before on to the next would work too. This was the simplest route once we discovered 4 calls were required. Shall I try doing each motor 4 times individually?

Reply:
Seems to work either way